### ISSUE TITLE
Update unofficial MacOS binaries

### ISSUE THREAD

**Issue by luzpaz** [mpv-player/mpv#7116]
Does anyone know djinn's github username https://laboratory.stolendata.net/~djinn/mpv_osx/ so we can ping him here to update to v0.30.0 ?

**Comment by Akemi** [comment]
@stolendata

**Comment by stolendata** [comment]
@luzpaz I'm a bit swamped with work, but it's coming.

**Comment by luzpaz** [comment]
@stolendata no worries, just ping me when it's available and I'll update homebrew cask

**Comment by stolendata** [comment]
@luzpaz It's available. As of 0.30.0 I will be including the man page in the archive, if you'd like to make use of Homebrew's new `manpage` artifact.

**Comment by luzpaz** [comment]
I'm actually submitting to homebrew-cask not homebrew

**Comment by Zegnat** [comment]
@luzpaz the manpage artifact was added based on a discussion that was had in homebrew-cask. (History: Homebrew/homebrew-cask#70869 → Homebrew/brew#6629 → Homebrew/homebrew-cask#71393.)

I actually emailed @stolendata asking if he could start including the manpages specifically for use with this. Though getting it with a release today was a bit sooner than I anticipated. I do not think the artifact is fully live just yet. Worth keeping in mind though, as it will make it a lot easier to use CLI mpv from a Cask install once manpages can automatically be assigned :)
